test/test_unistd.rs

```diff
@@ -29,23 +29,26 @@ fn test_fork_and_waitpid() {
                 Ok(WaitStatus::Exited(pid_t, _)) =>  assert!(pid_t == child),
 
                 // panic, must never happen
-                Ok(_) => panic!("Child still alive, should never happen"),
+                s @ Ok(_) => panic!("Child exited {:?}, should never happen", s),
```

Comment:
Okay, well if it isn't unreachable, then we definitely shouldn't use that macro!